appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: 0.10.{build}
+version: 1.0.{build}

```

Comment:
I wonder what does this version do? Should we mention in the release docs that it should be updated with each release? (if yes, then please add a line into issue #2639)

Reply:
Frankly, I have no idea :) It was there and I just made it look closer to what we call the version. However, this is not the only place where one can set up the version (there is also a field in the project settings). It would be nice to understand what it means, but in reality we don't need any of those... the ideal solution is just to drop it everywhere, where it makes no sense for us.